plugins/field-date/src/field_date.ts

```diff
@@ -55,18 +55,20 @@ export class FieldDate extends Blockly.FieldTextInput {
   /**
    * Ensures that the input value is a valid date.
    * @param newValue The input value. Ex: '2023-04-28'
```

Comment:
Can you add some additional info here stating that date strings including time zones will be converted ti ISO strings in the user's local time zone? I think this is just slightly surprising (even though it makes sense) so I would like to have it in English as well as code!

Reply:
Done!